code/game/gamemodes/gameticker.dm

```diff
@@ -510,6 +510,12 @@ var/datum/controller/gameticker/ticker
 	minesweeper_best_players["intermediate"] = "none"
 	minesweeper_best_players["expert"] = "none"
 
+/datum/controller/gameticker/proc/get_key(mob/M)
```

Comment:
This doesn't need to be a gameticker-level proc. Move it to _helpers/unsorted and remove the /datum/controller/gameticker bit from the front.